WordPress/Classes/ViewRelated/QR Login/View Controllers/QRLoginVerifyAuthorizationViewController.swift

```diff
@@ -196,9 +196,22 @@ extension QRLoginVerifyAuthorizationViewController {
 
         enum completed {
             static let imageName = "domains-success"
-            static let title = NSLocalizedString("You're logged in!", comment: "Title for the success view when the user has successfully logged in")
-            static let subtitle = NSLocalizedString("Tap dismiss and head back to your web browser to continue.", comment: "Subtitle instructing the user to tap the dismiss button to leave the log in flow")
-            static let confirmButton = NSLocalizedString("Dismiss", comment: "Button label that dismisses the qr log in flow and returns the user back to the previous screen")
+            static let title = NSLocalizedString(
+                "qrLoginVerifyAuthorization.completedInstructions.title",
+                value: "You're logged in!",
+                comment: "Title for the success view when the user has successfully logged in"
+            )
+            private static let subtitleFormat = NSLocalizedString(
+                "qrLoginVerifyAuthorization.completedInstructions.subtitle",
+                value: "Tap '%@' and head back to your web browser to continue.",
+                comment: "Subtitle instructing the user to tap the dismiss button to leave the log in flow. %@ is a placeholder for the dismiss button name."
+            )
+            static let confirmButton = NSLocalizedString(
+                "qrLoginVerifyAuthorization.completedInstructions.dismiss",
+                value: "Dismiss",
+                comment: "Button label that dismisses the qr log in flow and returns the user back to the previous screen"
+            )
+            static let subtitle = String(format: subtitleFormat, Self.confirmButton)
```

Comment:
See my comment in the commit: https://github.com/wordpress-mobile/WordPress-iOS/commit/ea70a120e71b93958f500b7c406a29bf04956053#comments